sds_data_manager/lambda_code/SDSCode/query_api.py

```diff
@@ -81,19 +82,23 @@ def lambda_handler(event, context):
 
     engine = db.get_engine()
     with Session(engine) as session:
-        search_result = session.execute(query).all()
+        search_results = session.execute(query).all()
+
+    # Convert the search results (list of tuples) to a list of dicts
+    search_results = [result._asdict() for result in search_results]
```

Comment:
Do we want to return everything from the database, or a limited subset of entries? I don't have a preference either way, so totally up to others if we want to trim even more columns/attributes off before returning to users.